client/hosting-overview/components/quick-actions-card.tsx

```diff
@@ -33,15 +34,18 @@ const Action: FC< ActionProps > = ( { icon, href, text } ) => {
 };
 
 const QuickActionsCard: FC = () => {
+	const translate = useTranslate();
 	const hasEnTranslation = useHasEnTranslation();
 	const site = useSelector( getSelectedSite );
+	const siteEditorUrl = useSelector( ( state: object ) =>
```

Comment:
Nit: can we put this inside the `useSelector()` group below together with the other URLs instead?

Reply:
Sure, I originally put it separated to avoid casting state as object 🙈